src/com/sap/piper/BashUtils.groovy

```diff
@@ -8,6 +8,9 @@ class BashUtils implements Serializable {
      * Put string in single quotes and escape contained single quotes by putting them into a double quoted string
      */
     static String quoteAndEscape(String str) {
+        if(str == null) {
+            return 'null'
+        }
```

Comment:
I just want to make sure that I am not breaking something with my changes.

If you write `"${str}` for `str == null` you get a string "null". So I emulate this behavior for `"${q(str)}"`. Otherwise the replacing below would cause an NPE.